FSIM_USAGE.md

```diff
@@ -0,0 +1,170 @@
+# FIDO Service Info Module (FSIM) Usage Guide
+
+This guide explains how to use the standard FIDO Service Info Modules (FSIMs) with the go-fdo-server owner service. FSIMs enable device onboarding with automated file transfers and command execution during the TO2 protocol phase.
+
+## Overview
+
+The go-fdo-server owner service supports four standard FSIMs:
+
+- **fdo.command** - Execute commands on the device
+- **fdo.download** - Download files from owner to device  
+- **fdo.upload** - Upload files from device to owner
+- **fdo.wget** - Have device download files from URLs
+
+FSIMs are activated by adding the corresponding command-line flags when starting the owner service.
+
+## Prerequisites
+
+- FDO server setup completed (see main README.md)
+- Owner service configured with proper certificates and keys
+- Device successfully initialized and voucher transferred to owner
+
+## fdo.command FSIM
+
+### Purpose
+Execute shell commands on the device during onboarding.
```

Comment:
On the device side in order for this to work the command line option --echo-commands needs to be provided:

`go-fdo-client ... --echo-commands`